sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -81,7 +81,15 @@
      }
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: C:\\Users\\nirovins\\OneDrive - Microsoft\\Documents\\AzureML\\config.json\n"
+     ]
+    }
+   ],
```

Comment:
Do we want to keep the contents in the output cells? If not, please clean them.

Reply:
Fixed